bot/exts/evergreen/emoji_count.py

```diff
@@ -17,73 +19,71 @@ class EmojiCount(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    def embed_builder(self, emoji: dict) -> discord.Embed:
+    @staticmethod
+    def embed_builder(emoji: dict) -> Tuple[discord.Embed, str]:
```

Comment:
this should be
```suggestion
    def embed_builder(emoji: dict) -> Tuple[discord.Embed, List[str]]:
```
make sure to import `List` from `Typing`